schema.sql

```diff
@@ -0,0 +1,48 @@
+/* Database schema to keep the structure of entire database. */
+
+CREATE TABLE animals (
+ID INT,
+NAME  VARCHAR,
+DATE_OF_BIRTH DATE,
+ESCAPE_ATTEMPTS INT,
+NEUTERED BOOLEAN,
+WEIGHT_KG DECIMAL
+Species VARCHAR 
+);
+
+CREATE TABLE owners(
+id  SERIAL PRIMARY KEY,
+full_name VARCHAR(50),
+age int
+);
+CREATE TABLE
+
+CREATE TABLE species(
+id  SERIAL PRIMARY KEY,
+name VARCHAR(50)
+);
+CREATE TABLE
+
```

Comment:
- Please, kindly remove the unfinished statement as its not being used in your codebase to also, reduce any form of confusion to keep your codebase orgnaized.